docs/Documentation/Applications/wrf.md

```diff
@@ -1 +1,665 @@
+# How to use the WRF application software 
+
+**Documentation:** [Weather Research Framework (WRF) Model](https://www.mmm.ucar.edu/models/wrf)
+
+*The [WRF](https://www.mmm.ucar.edu/models/wrf) model is a state of the art mesoscale numerical weather prediction system designed for both atmospheric research and operational forecasting applications.*
+
+## Getting Started
+
+This section provides the minimum amount of information necessary to successfully run a WRF job on an NREL Cluster. First, we show how to use WRF given that we have different versions of WRF already built and available as modules.
+
+
+```
+% module avail wrf
+     wrf/4.1.3/intel-20.1.217-mpi (D)    wrf/4.2.1/intel-20.1.217-mpi
+     
+```
+
+The `module avail wrf` command shows that two wrf modules are
+available for two different versions of wrf and built with the intel
+compiler v20.1 toolchain. This command also shows that the version
+4.1.3 is the default build which is automatically loaded with `module
+load wrf`. Users are free to choose any of the module versions
+available for use. Currently there are no modules to run wrf on GPUs, but
+there is current effort underway to make that available on future systems.
+
+Next, we look at how to use the wrf module. Below is an example job script
+
+### Example Job Scripts
+
+??? example "Kestrel CPU"
+
+	```slurm
+	#!/bin/bash
+
+	# In a comment summarize the hardware requested, e.g. number of nodes, 
```

Comment:
The template text comment can be removed, but if you have recommendations for hardware or other sbatch directives, that could be included in a comment here